spec/index.md

```diff
@@ -427,6 +427,16 @@ The state tree contains information about the topology of the Internet Computer.
         principal = bytes .size (0..29)
         tagged<t> = #6.55799(t) ; the CBOR tag
 
+-   `/subnet/<subnet_id>/metrics` (blob)
```

Comment:
A danger of this endpoint is that replica-signed queries require read_state requests to `/subnet` and thus its responses would now also serve the metrics (not a problem is we don't have many metrics, but could be a problem in the future).

Reply:
I am not sure I'm following what's the problem. I suppose for replica-signed queries you don't strictly need the metrics but the metrics are also not a secret, so returning them seems ok security wise.

Reply:
It is not about security but about data size. If the subnets contain all public keys of the nodes hosting the subnet, though, that that is a lot more data (~ 1KB per subnet) so the metrics at this point don't really add a lot of data on top of this.

Reply:
I see, thanks for the clarification. Yeah, I don't expect the metrics will ever be anything other than a few counters, so something in the order of hundreds of bytes extra in the worst case.